linux/keyman-config/keyman_config/__init__.py

```diff
@@ -55,7 +55,7 @@ def _(txt):
 
         sentry_logging = LoggingIntegration(
             level=logging.INFO,           # Capture info and above as breadcrumbs
-            event_level=logging.CRITICAL  # Send critical errors as events
+            event_level=None              # Don't create events for errors
```

Comment:
Won't this mean all no events are sent to sentry even unhandled events?

Reply:
What was I thinking??? Of course you're right.

Reply:
done.